index.html

```diff
@@ -54,41 +56,46 @@ <h3 class="h3-first">I'm a Software Developer</h3>
                 </ul>
             </div>
         </section>
-        <section class="content">
-            <ul class="projects-grid">
-                <li class="grid-item recent-works-outer">
+        <section class="content grid-container-outer">
+            <ul class="content grid-container">
+                <li class="grid-item recent-works-outer grid-item-top-pc new-recent-works-outer">
                     <h2 class="recent-works">
                         <span>My Recent Works</span>
                     </h2>
+                    <hr class="mobile-hide recent-works-line">
                 </li>
-                <li class="grid-item multi-post-stories">
-                    <a href=""><img src="Icons/img-placeholder1.png" alt=""></a>
-                    <h3 class="multi-post-title h3-second">Multi-Post Stories</h3>
-                    <p class="multi-post-text">
-                        A daily selection of privately personalized reads; no accounts or sign-ups required; 
-                        has been the industry's standard dummy text ever since the 1500s, when an unknown 
-                        printer took a standard dummy text.
-                    </p>
-                    <ul class="language-btns-container">
-                        <li>
-                            <a href="" class="language-btn">CSS</a>
-                        </li>
-                        <li>
-                            <a href="" class="language-btn">HTML</a>
-                        </li>
-                        <li>
-                            <a href="" class="language-btn">Bootstrap</a>
-                        </li>
-                        <li>
-                            <a href="" class="language-btn">Ruby</a>
-                        </li>
-                    </ul>
-                    <a href="" class="see-project-outer">
-                        <div class="see-project">
-                            See Project
-                        </div>
-                    </a>
+                <li class="grid-item multi-post-stories-outer grid-item-top-pc">
+                    <a href="" class="multi-post-image"><img src="Icons/img-placeholder1.png" alt=""></a>
+                    <div class="multi-post-stories multi-post-stories-column">
+                        <h3 class="multi-post-title h3-second">Multi-Post Stories</h3>
+                        <p class="multi-post-text">
+                            A daily selection of privately personalized reads; no accounts or sign-ups required; 
+                            has been the industry's standard dummy text ever since the 1500s, when an unknown 
+                            printer took a standard dummy text.
+                        </p>
```

Comment:
- [ ] You did a great job on the styles but for this project, you are supposed to stick to the [Figma ](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471)template as much as possible. Please reduce the gap between (image and details) these elements and make sure it matches the design on the template.

### Your design

![image](https://user-images.githubusercontent.com/83803414/160229779-8c22b521-ed04-4afe-8db5-2ed029569c5b.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/160229788-cb7a7776-321e-4295-9888-633668e53e40.png)